src/backend/marsha/core/models/video.py

```diff
@@ -245,6 +255,68 @@ def get_mediapackage_endpoints(self):
         """Return the mediapackage enspoints info."""
         return self.live_info.get("mediapackage").get("endpoints")
 
+    def get_recording_slices(self):
+        """Return empty array if no recording slices exists."""
+        if not self.recording_slices:
```

Comment:
maybe we could use a default value in the model ? default=[]

Reply:
I've tried it, but it didin't worked.
I'll give it another shot.